HW-11/Homework-11.py

```diff
@@ -0,0 +1,60 @@
+# decorator for any function:
+def my_new_decorator(func):
+    def the_wrapper_around_the_original_function(*args, **kwargs):
+        from datetime import datetime
+        print(func.__name__, datetime.now().strftime('%H:%M'))
+        func(*args, **kwargs)
```

Comment:
It is better to use `return` statement here to be able to decorate functions that return some value